api/src/main/java/org/mandulis/mts/dto/AttachmentDto.java

```diff
@@ -0,0 +1,15 @@
+package org.mandulis.mts.dto;
+
+import java.io.InputStream;
+
+public class AttachmentDto {
```

Comment:
you'll need getters and setters if you plan to use this DTO

Reply:
There are public props, why add getters setters?

Reply:
We don't use props, you can import lombok and just annotate it with @Getter and @Setter

Reply:
Oki doki, added